examples/crucible.py

```diff
@@ -0,0 +1,228 @@
+import asyncio
+import os
+import textwrap
+import typing as t
+
+import click
+import requests
+from loguru import logger
+from pydantic import StringConstraints
+
+import rigging as rg
+
+logger.remove()
+logger.add(
+    lambda msg: print(msg, flush=True),
+    format="<cyan>{time:YYYY-MM-DD HH:mm:ss.SSS}</cyan> | <level>{level: <8}</level> | <blue>{name}:{function}:{line}</blue> - {message}",
+    colorize=True,
+)
```

Comment:
No big deal, wonder if there is a reason to not use the logging mechanics from rigging to set all this up

Reply:
more than a very valid call, apologies and something i missed in the docs! (which are awesome) :) fixed and retested in [chore: use native rigging logger support](https://github.com/dreadnode/rigging/pull/20/commits/432ef7777e9f8c9d9600baa85016d21c42962f72)